kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/position/PositionService.java

```diff
@@ -112,6 +113,15 @@ public interface PositionService {
     @Deprecated
     public String getLastSentence();
 
+    /**
+     * Returns the GNSS System used to get the position information. Could be one or more system (eg: GPS + GLONASS).
```

Comment:
Add a reference to the `enum` so that whoevers read the docs can refer to it easily. Look up the Javadoc syntax, I'm going from memory.

```suggestion
     * Returns the GNSS System used to get the position information @reference GNSSType. Could be one or more system (eg: GPS + GLONASS).
```